### PR TITLE
update wiki Bootstrap story #5239

### LINKED ISSUE #5239: Missing addition of bootstrap.min.js to the scripts array in with bootstrap wiki-story
I've just discovered, that some parts of bootstrap aren't usable after following the guide given in the corresponding [wiki article](https://github.com/angular/angular-cli/wiki/stories-include-bootstrap).

More clearly, an alert I've added, wasn't closed after pressing the corresponding "x" link in the alert div.
My guess is, that the addition of `bootstrap.min.js` to the scripts array inside `.angular-cli.json` is missing in the wiki article.

### PR DESCRIPTION
**Pull request by GoNode5**
Fix #5239

### CODE REVIEW

File: docs/documentation/stories/include-bootstrap.md [hunk 1]
```diff
@@ -44,7 +44,27 @@ Now that the project is set up it must be configured to include the bootstrap CS
     "../node_modules/bootstrap/dist/css/bootstrap.min.css",
     "styles.css"
   ],
-  ```
+
```
Comment (delasteve) [inline docs/documentation/stories/include-bootstrap.md]: Add ` ``` ` here.

File: docs/documentation/stories/include-bootstrap.md [hunk 2]
```diff
@@ -44,7 +44,27 @@ Now that the project is set up it must be configured to include the bootstrap CS
...
+  to see which versions of jQuery are supported.  
+  https://v4-alpha.getbootstrap.com/getting-started/javascript/
+  
+  ``json
+  "scripts": [
+  	"../node_modules/jquery/dist/jquery.js",
+		"../node_modules/tether/dist/js/tether.js",
+		"../node_modules/bootstrap/dist/js/bootstrap.js",
+  ]
+
```
Comment (delasteve) [inline docs/documentation/stories/include-bootstrap.md]: Add ` ``` ` here

File: docs/documentation/stories/include-bootstrap.md [hunk 3]
```diff
@@ -44,7 +44,27 @@ Now that the project is set up it must be configured to include the bootstrap CS
...
+  **Note:** from Bootstrap site 
+  
+  Some plugins and CSS components depend on other plugins. If you include plugins individually, 
+  make sure to check for these dependencies in the docs. Also note that all plugins depend on 
+  jQuery (this means jQuery must be included before the plugin files). Consult our bower.json 
+  to see which versions of jQuery are supported.  
+  https://v4-alpha.getbootstrap.com/getting-started/javascript/
+  
+  ``json
```
Comment (delasteve) [inline docs/documentation/stories/include-bootstrap.md]: Forgot a <code>`</code> here.

**Comment by delasteve** [comment]
There's another section at the bottom that references `version 3` and `version 4`. While your in that file, would you mind making those inline with the others and make them `version 3.x` and `version 4.x`?

Thanks!

File: docs/documentation/stories/include-bootstrap.md (hunk 2 above)
Comment (GoNode5) [inline docs/documentation/stories/include-bootstrap.md]: My first wiki pull apperently. Will try again. Thanks for your help.

**Comment by clydin** [comment]
If this is included, a note pointing to some of the angular bootstrap projects should also be included.  As they would be most likely preferred by many users.

* https://ng-bootstrap.github.io
* http://valor-software.com/ng2-bootstrap

**Comment by GoNode5** [comment]
These are individual projects/component  that  have anything todo with the setup of Angular-cli. These are specific settings to get Angular-cli working with Bootstrap.

**Comment by clydin** [comment]
Those are alternatives to the bootstrap javascript that provide angular friendly directives/components.

**Comment by Brocco** [comment]
I tend to agree with @clydin that most people will prefer one of the Angular specific implementations of bootstrap's components.

If this is a route that you would like to go down, I would suggest creating a new story 

> Using Bootstrap Components

That covers these 3 scenarios:

- ng bootstrap https://ng-bootstrap.github.io
- ng2 bootstrap http://valor-software.com/ng2-bootstrap/
- bootstraps jQuery-based libraries

Then link to this story as part of the existing bootstrap setup story.

**Comment by Brocco** [comment]
I've been thinking about this a little more... I'm OK with adding JS-based logic to this document directly, but I still feel strongly that we should present all 3 options mentioned above.

The point of this document is providing a way for people to setup Bootstrap, not to be opinionated about the means in which they set it up. Can you please add implementation instructions for the other two options? As well as a brief paragraph above the 3 explaining what the options are in an unbiased manner?

**Comment by GoNode5** [comment]
The question was to add a note about  how to include the basic Bootstrap JS, because Bootstrap would not always work as aspected. In case of  https://ng-bootstrap.github.io, you still need to add the css but not the JS files (but I have them both and seems to work). In the whole it becoming more complicated than the original remarks. 
 The third parties should instruct how to install in combination with Angular-cli. I don't have a bias but my experience with  https://ng-bootstrap.github.io that is missed a lot of options and still alpha. That is way I'am a bit hesistant. It could also become a longer list to maintain. Maybe an 'works with Angular-cli component list'?

**Comment by filipesilva** [comment]
@Brocco can you re-review please?

File: docs/documentation/stories/include-bootstrap.md [hunk 4]
```diff
@@ -45,9 +45,41 @@ Now that the project is set up it must be configured to include the bootstrap CS
...
+  ]
+```
 **Note:** When you make changes to `.angular-cli.json` you will need to re-start `ng serve` to pick up configuration changes.
 
+### Other Bootstrap component libraries
+
+ - ng bootstrap https://ng-bootstrap.github.io (version 4.x only)
+ - ng2 bootstrap http://valor-software.com/ng2-bootstrap/ (version 3.x & 4.x)
+ 
+You should check with the libraries for instructions how to include their project with Angular-cli. You probably only need to
```
Comment (Brocco) [inline docs/documentation/stories/include-bootstrap.md]: If you're not going to provide specific instructions for each of these solutions, please remove this line and leave just the links.

File: docs/documentation/stories/include-bootstrap.md [hunk 5]
```diff
@@ -45,9 +45,41 @@ Now that the project is set up it must be configured to include the bootstrap CS
...
+make sure to check for these dependencies in the docs. Also note that all plugins depend on 
+jQuery (this means jQuery must be included before the plugin files).
+Consult our bower.json to see which versions of jQuery are supported.
+https://v4-alpha.getbootstrap.com/getting-started/javascript/
+
+In `.angular-cli.json` add the following lines to the scripts section:
+
+# version 4.x
+  "scripts": [
+  	"../node_modules/jquery/dist/jquery.js",
```
Comment (Brocco) [inline docs/documentation/stories/include-bootstrap.md]: Please make the spacing consistent (this is a minor nit-pick, but I've also requested other changes 😀)

**Pull request closed without merging**